desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/MutableOfferView.java

```diff
@@ -258,7 +258,7 @@ protected void doActivate() {
 
             currencyComboBox.getSelectionModel().select(model.getTradeCurrency());
             paymentAccountsComboBox.setItems(getPaymentAccounts());
-            UserThread.execute(() -> paymentAccountsComboBox.getSelectionModel().select(model.getPaymentAccount()));
+            paymentAccountsComboBox.getSelectionModel().select(model.getPaymentAccount());
```

Comment:
Did this change avoid that issue? The UserThread.execute schedules the execution to the next render frame and serves as a small delay which often fixed UI synchronisation issues.